src/app/api/help/[id]/service.js

```diff
@@ -0,0 +1,23 @@
+import { updateFieldInDocument } from '../../../../utils/firebase/firestore/updateFieldInDocument.js';
+import getDocument from '../../../../utils/firebase/firestore/getDocument.js';
+import { arrayUnion } from "firebase/firestore";
+
+const updateFieldInHelp = updateFieldInDocument('helps');
+
+export async function getHelpById(id) {
+    const helpData = await getDocument('helps', id)
+    return { docId: helpData.result.id, ...helpData.result.data() };
```

Comment:
make sure the helpData exist. you can add ? 
or get result
 const { result } =

Reply:
okey